.ci/ci-fast-return.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2017-2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Perform CI 'fast path return' checks.
+# Returns: 1 for 'fast return OK', 0 for 'do not fastpath'.
```

Comment:
A minor point, but the name of the script suggests to me that "success" means fast path. But in that case, `0` should mean "yes, fast path" and non-zero should therefore mean "nope, run the whole CI".

Reply:
done, I think.

Reply:
hah, yep. see, comments always rot... updated....